api/control.proto

```diff
@@ -317,10 +317,15 @@ message JoinTokenRotation {
 	// RotateWorkerToken tells UpdateCluster to rotate the worker secret.
 	bool rotate_worker_token = 1;
 
-	// RotateManagerSecret tells UpdateCluster to rotate the manager secret.
+	// RotateManagerToken tells UpdateCluster to rotate the manager secret.
 	bool rotate_manager_token = 2;
 }
 
+message UnlockKeyRotation {
```

Comment:
To be less disruptive, let's rename the message `KeyRotation` and have a bool for each key to be rotation.